apps/web-giddh/src/app/daybook/daybook.component.html

```diff
@@ -14,7 +14,7 @@
 
             <div class="form-inline d-block flex-row flex-nowrap">
                 <div class="form-group custom-datepicker" (click)="showGiddhDatepicker($event)">
-                    <i><img src="assets/images/custom-calender.svg"></i>
+                    <i class="icon-calender calender-addone"></i>
```

Comment:
correct spelling of icon-calender -> icon-calendar and calender-addone -> calendar addon @shalinee-walkover